### PR TITLE
add drink carousel on mobile

### PR DESCRIPTION
**Pull request by SimonLab**
ref: https://github.com/club-soda/club-soda-guide/issues/465#issuecomment-463229609

- display carousel on mobile for drinks
- display chevrons

### CODE REVIEW

File: assets/elm/DrinksCarousel.elm [hunk 1]
```diff
@@ -129,12 +129,12 @@ subscriptions model =
 view : Model -> Html Msg
 view model =
     div [ class "relative" ]
-        [ img [ src "images/up-chevron.svg", alt "left arrow", class "dn db-ns f1 b pointer absolute-vertical-center left-2 rotate-270 h1", onClick CarouselLeft ] []
+        [ img [ src "images/up-chevron.svg", alt "left arrow", class "z-max f1 b pointer absolute-vertical-center left-1 left-2-ns rotate-270 h1", onClick CarouselLeft ] []
```
Comment (Cleop) [inline assets/elm/DrinksCarousel.elm]:
```suggestion
        [ img [ src "images/up-chevron.svg", alt "left arrow", class "z-1 f1 b pointer absolute-vertical-center left-1 left-2-ns rotate-270 h1", onClick CarouselLeft ] []
```

File: assets/elm/DrinksCarousel.elm [hunk 2]
```diff
@@ -129,12 +129,12 @@ subscriptions model =
...
         , div [ class "flex-ns flex-wrap w-90 center pv4-ns dn dib-ns" ]
             (renderDrinksCarousel model 3)
         , div [ class "flex-wrap w-90 center db dn-ns", id "carousel" ]
-            [ renderDrinksCarouselMobile model ]
-        , img [ src "images/up-chevron.svg", alt "right arrow", onClick CarouselRight, class "dn db-ns f1 b pointer absolute-vertical-center right-2 rotate-90 h1" ] []
+            (renderDrinksCarouselMobile model)
+        , img [ src "images/up-chevron.svg", alt "right arrow", onClick CarouselRight, class "z-max f1 b pointer absolute-vertical-center right-1 right-2-ns rotate-90 h1" ] []
```
Comment (Cleop) [inline assets/elm/DrinksCarousel.elm]:
```suggestion
        , img [ src "images/up-chevron.svg", alt "right arrow", onClick CarouselRight, class "z-1 f1 b pointer absolute-vertical-center right-1 right-2-ns rotate-90 h1" ] []
```

File: assets/elm/DrinksCarousel.elm (hunk 1 above)
Comment (Cleop) [reply assets/elm/DrinksCarousel.elm]: This still appears as it should with `z-1` and it's better practice to only sent a z-index as high as you need to, so I'd favour z-1 over z-max where possible.

**Comment by Cleop** [comment]
Your existing version is fine but there's a change Jussi may prefer the arrows off the cards. So you may want to ask...

Your existing version:


Alternative version:


You can get the arrows positioned like this if you change their existing:
`right-1` /`left-1` aka `right: 1rem;` or `left: 1rem;` to be `right: -0.5rem` and `left: -0.5rem`.

I think you'd need to create new classes for those and make sure it's not impacting the desktop version.

**Comment by SimonLab** [comment]
I've updated the PR based on your comments @Cleop , could you check it again? thanks

**Comment by Cleop** [comment]
Looks good now on mobile @SimonLab however the chevrons don't look right on tablet / desktop (I think they're using the same css so hopefully only one thing needs fixing). It's hard to see because you can't see the edge of my screen in the screenshots but compare it with the width of my navbar and you can see the chevrons are at the very edge and being cut off slightly:

Tablet:
image

Desktop:
image

Desktop on staging currently:
image

**Comment by SimonLab** [comment]
thanks for spotting this @Cleop It should be fixed now as I apply the specific classes for the chevrons only on mobile now

**Comment by Cleop** [comment]
I'm still getting the same look on tablet... mobile and desktop look good though 👍

**Comment by SimonLab** [comment]
@Cleop I've added a media query to only apply the `left-0-5` and `right-0-5` on mobile, ie 60rem:
```
@media (max-width: 60em ) {
  .left-0-5 {
    left: -0.5rem;
  }

  .right-0-5 {
    right: -0.5rem;
  }
}
```

So is this media query also applies to table? If not then the chevrons have the `left-2` and `right-2` applied on desktop and tablet.

Do you know if I need to change this media query value?

**Comment by Cleop** [comment]
48em is where media queries start for tablets according to Tachyons. So if we need the same styling on desktop and tablet then it looks like you need to change the 60em to 48em.

**Comment by Cleop** [comment]
@SimonLab apologies, this still doesn't look right on tablet. I've inspected and played with and tablet needs its own breakpoint to mobile/desktop. Rather than 2rem or -0.5rem, tablet need left and right set to `0.5rem`. As for the breakpoint, an ipad is 768px which is equivalent to 48em, so the breakpoint should be from 48em to 64em (as tachyons suggests).

**Pull request merged**